### PR TITLE
Update a Simplified Chinese translation file.

### PR DESCRIPTION
**Pull request by njlyf2011**

### CODE REVIEW

File: dll/win32/shell32/lang/zh-CN.rc [hunk 1]
```diff
@@ -5,6 +5,7 @@
  * PURPOSE:     Chinese (Simplified) Language Resources
  * TRANSLATORS: 2008 zhangbing <e_zb@21cn.com, ezb@mail.gywb.cn>
                 2011 Elton Chung aka MfldElton <elton328@gmail.com>
+				2017 Luo Yufan <njlyf2011@hotmail.com>
```
Comment (HBelusca) [inline dll/win32/shell32/lang/zh-CN.rc]: Please use spaces and not tabs for your  indentation.

File: dll/win32/shell32/lang/zh-CN.rc [hunk 2]
```diff
@@ -848,23 +848,22 @@ BEGIN
...
+    IDS_ADVANCED_HIDDEN "隐藏的文件和文件夹"
+    IDS_ADVANCED_DONT_SHOW_HIDDEN "不显示隐藏的文件和文件夹"
+    IDS_ADVANCED_SHOW_HIDDEN "显示隐藏的文件和文件夹"
+    IDS_ADVANCED_HIDE_FILE_EXT "隐藏已知文件类型的拓展名"
+    IDS_ADVANCED_SUPER_HIDDEN "隐藏受保护的操作系统文件 (推荐)"
+    IDS_ADVANCED_DESKTOP_PROCESS "在单独的进程中打开文件夹窗口"
+    IDS_ADVANCED_CLASSIC_VIEW_STATE "记住每个文件夹的视图设置"
+    IDS_ADVANCED_PERSIST_BROWSERS "在登录时还原上一个文件夹窗口"
+    IDS_ADVANCED_CONTROL_PANEL_IN_MY_COMPUTER "在我的电脑中显示控制面板"
+    IDS_ADVANCED_SHOW_INFO_TIP "鼠标指向文件夹和桌面项时显示提示信息"
```
Comment (HBelusca) [inline dll/win32/shell32/lang/zh-CN.rc]: IDS_ADVANCED_SHOW_COMP_COLOR disappeared?

File: dll/win32/shell32/lang/zh-CN.rc [hunk 3]
```diff
@@ -807,7 +808,7 @@ BEGIN
     IDS_ICO_FILE "图标"
     IDS_INI_FILE "配置设置"
     IDS_LNK_FILE "快捷方式"
-    IDS_PIF_FILE "NT VDM 程序信息文件"
+    IDS_PIF_FILE "NTVDM 程序信息文件"
```
Comment (HBelusca) [inline dll/win32/shell32/lang/zh-CN.rc]: "NT VDM" was correct.

**Comment by njlyf2011** [comment]
Fixed.

**Comment by HBelusca** [comment]
Why closing? The code hasn't been committed yet! Are you going to restart a new translation PR soon?

**Comment by njlyf2011** [comment]
There is something wrong with the translate file.
So I closed it.

**Pull request closed without merging**